pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -135,10 +135,20 @@ func (l *listUsersQuery) ListUsers(
 // 	return nil
 // }
 
-func (l *listUsersQuery) expand(
+func (l *listUsersQuery) beginExpand(
```

Comment:
Can we consolidate to a single recursive function? This keeps it consistent with Check and doesn't introduce another function. Can just initialize `VisitedUsersets` if empty.